scss/_list-group.scss

```diff
@@ -12,6 +12,17 @@
   @include border-radius($list-group-border-radius);
 }
 
+.list-group-numbered {
```

Comment:
Need to get feedback from brand for numbered list-group https://deploy-preview-650--boosted.netlify.app/docs/5.0/components/list-group/#numbered

Reply:
will see later